docker-compose.yml

```diff
@@ -57,16 +63,25 @@ services:
     depends_on: [keycloak]
 
   # #############################################################################################
-  # ###                                  KEYCLOAK CONFIG                                      ###
+  # ###                                  SEQ CONFIG                                           ###
   # #############################################################################################
   seq:
     image: datalust/seq
     environment:
       - ACCEPT_EULA=Y
     ports:
-      - 5341:5341
-      - 8041:80
+      #- 5341:5341   # logs are written here, do not need to expose
+      - 8083:80      # user interface is available here
+
+  # #############################################################################################
+  # ###                                  ZIPKIN CONFIG                                        ###
+  # #############################################################################################
+
+  zipkin:
```

Comment:
zipkin is only used in development right now. May be useful to use Jaeger in production, but not required.